### PR TITLE
feat(registry): add support for project in namespace

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

**Review by Sh4d1 — CHANGES_REQUESTED** [review]
Let's wait, it's not deployed yet

**Review by jerome-quere — APPROVED** [review]

File: docs/resources/registry_namespace.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ The following arguments are supported:
 
 - `organization_id` - (Defaults to [provider](../index.md#organization_id) `organization_id`) The ID of the organization the registry is associated with.
 
+- `project_id` - (Defaults to [provider](../index.md#project_id) `project_id`) The ID of the project the registry is associated with.
```
Comment (QuentinBrosse) [inline docs/resources/registry_namespace.md:39]: NIT: in the data-source, you use `registry` and in the resource your use `registry`, we should choose one word.

**Review by QuentinBrosse — DISMISSED** [review]
LGTM

File: scaleway/resource_registry_namespace.go [hunk 2]
```diff
@@ -96,8 +94,6 @@ func resourceScalewayRegistryNamespaceRead(ctx context.Context, d *schema.Resour
 
 	_ = d.Set("name", ns.Name)
 	_ = d.Set("description", ns.Description)
-	// Registry does not support yet project id.
-	// Once it does, use ProjectID instead or OrganizationID
```
Comment (Sh4d1) [inline scaleway/resource_registry_namespace.go:100]: set project id -> project id and org id -> org id

**Review by Sh4d1 — CHANGES_REQUESTED** [review]

**Review by Sh4d1 — APPROVED** [review]
LGTM

**Pull request merged**